<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1077</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1077</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(247,240,247);
font-size: 12px;
font-family: &quot;System&quot;;
color: rgb(0, 0, 0);
}

QLineEdit
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}

QTextBrowser
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}

QPushButton
{
background-color: rgb(208,211,246);
border-radius: 8px;
color: rgb(0, 0, 0);
}

QPushButton:hover
{
	background-color: rgb(176, 170, 255);
}

QPushButton:pressed
{
	background-color: rgb(152, 147, 220);
}


QMenu::item:selected 
{
    background-color: rgb(176, 170, 255);

}

QRadioButton::indicator::unchecked {
   background-color: rgb(208,211,246);
}

QRadioButton::indicator:unchecked:hover {
   background-color: rgb(176, 170, 255);
}

QRadioButton::indicator:unchecked:pressed {
    background-color: rgb(152, 147, 220);
}

QRadioButton::indicator::checked {
   background-color: rgb(152, 147, 220);
}

</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="txtBr_route">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_route">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
font-family: &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>route</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_lenRoute">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>460</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>route </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lenRoute">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
font-family: &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_girl">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/img/back73.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_alg">
     <item>
      <widget class="QLabel" name="label_choiceAlg">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>choose an algorithm:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_width">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>BFS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_depth">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>DFS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_Dijkstra">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Dijkstra's algorithm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pB_createGraph">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>create graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stripe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/img/back1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_route_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
font-family: &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_startVert">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_start_v_ok">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_drawGraph">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>draw</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>821</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid"/>
   </widget>
   <zorder>label_girl</zorder>
   <zorder>label_route</zorder>
   <zorder>txtBr_route</zorder>
   <zorder>label_lenRoute</zorder>
   <zorder>pB_lenRoute</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pB_createGraph</zorder>
   <zorder>label_stripe</zorder>
   <zorder>label_route_2</zorder>
   <zorder>lineEdit_startVert</zorder>
   <zorder>pB_start_v_ok</zorder>
   <zorder>pB_drawGraph</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_editGraph">
    <property name="title">
     <string>Edit graph</string>
    </property>
    <widget class="QMenu" name="menu_top">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>System</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Vertex                                       </string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/back7.jpg</normaloff>../resource/back7.jpg</iconset>
     </property>
     <property name="separatorsCollapsible">
      <bool>false</bool>
     </property>
     <addaction name="top_add"/>
     <addaction name="separator"/>
     <addaction name="top_del"/>
    </widget>
    <widget class="QMenu" name="menu_edge">
     <property name="title">
      <string>Edge</string>
     </property>
     <addaction name="edge_add"/>
     <addaction name="separator"/>
     <addaction name="edge_del"/>
     <addaction name="separator"/>
     <addaction name="edge_edit"/>
    </widget>
    <widget class="QMenu" name="menu_matrix">
     <property name="title">
      <string>Matrix</string>
     </property>
     <addaction name="matrix_adj_menu"/>
    </widget>
    <addaction name="menu_top"/>
    <addaction name="separator"/>
    <addaction name="menu_edge"/>
    <addaction name="separator"/>
    <addaction name="menu_matrix"/>
   </widget>
   <addaction name="menu_editGraph"/>
  </widget>
  <action name="top_add">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/circle.png</normaloff>:/rec/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Add                        </string>
   </property>
  </action>
  <action name="top_del">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/circle.png</normaloff>:/rec/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="edge_add">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/route.png</normaloff>:/rec/img/route.png</iconset>
   </property>
   <property name="text">
    <string>Add                          </string>
   </property>
  </action>
  <action name="edge_del">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/route.png</normaloff>:/rec/img/route.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="edge_edit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/route.png</normaloff>:/rec/img/route.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="matrix_adj_menu">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/matrix.png</normaloff>:/rec/img/matrix.png</iconset>
   </property>
   <property name="text">
    <string>Adjacency matrix</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
